src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -425,9 +426,23 @@ rct_window* WindowSceneryOpen()
     if (window != nullptr)
         return window;
 
+    // count scenery groups to determine window width
```

Comment:
Could you spin off this code into a static function of its own? `WindowSceneryGetWidth()`